### PR TITLE
Add docker

### PR DESCRIPTION
**Pull request by RoriCremer**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+module.exports = {
+  'plugins': ['lodash-fp', 'react'],
+  'extends': 'react-app',
+  'globals': {
+    'SATURN_VERSION': false,
```
Comment (ddietterich) [inline .eslintrc.js:5]: Will these eventually be JADE things?

**Review by ddietterich — COMMENTED** [review]
LGTM FWIW :)

File: .eslintrc.js (hunk 1 above)
Comment (thathert) [reply .eslintrc.js:5]: Yes, eventually once we remove the instance of SATURN_VERSION, etc from the code.

File: docker_push [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push USER/REPO
```
Comment (thathert) [inline docker_push]: I don't think we need this file. We can just update the .travis.yml and say `... && docker push "$DOCKER_USERNAME"/"$DOCKER_REPO"`

**Review by thathert — CHANGES_REQUESTED** [review]
Another minor change, looks good though!

**Pull request merged**